docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM node:12.16.1
+# NOTE: If editing this file then check if cleanup of .trivyignore is needed also!
+FROM node:12-buster
```

Comment:
We should specifically ask for `12.19.0-buster` to build reproducibility.

Reply:
We actually don't care about the version of Node. Eventually Martha's various endpoints ends up in GCF where we have no control over the minor version. Most likely (hopefully! 🤞) Google is constantly upgrading the GCFs to the latest Node point releases with security patches, etc.

So, these images aren't used to \*run\* the eventual deployments. They're used a) to emulate in a FiaB and b) later \*to\* deploy the GCF. (This image is quite strange and contains a lot.) Still, for consistency with the other Firecloud/Terra images that are used in production I'll freeze the minor version.

Reply:
Oh dear. Well, in that case, maybe we should be non-specific. I misunderstood the purpose of this Dockerfile.
When deploying the cloud function, do we at least control the major version?

Reply:
We do control the major deploy version, here:

https://github.com/broadinstitute/martha/blob/9277913f57c6d5b6da5b3e1debd9ff0520c830df/deploy.sh#L111-L117

As for how-the-sausage-is-made, you'll notice the above deploys actually on 10. Here's another inconsistency where we code against 12, which matches what we currently emulate on:

- https://github.com/broadinstitute/martha/blob/9277913f57c6d5b6da5b3e1debd9ff0520c830df/.node-version#L1

For the lurkers, relevant links (constantly updating):
- https://nodejs.org/en/about/releases/
- https://cloud.google.com/functions/docs/concepts/nodejs-runtime

Reply:
Ugh, I hate that it's so inconsistent, but that's not the purpose of this PR so I'll leave it alone for now.